### PR TITLE
[Api] Customer see description of a product

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Review by AdamKasp — APPROVED** [review]

File: src/Sylius/Behat/Page/Shop/Product/ShowPageInterface.php [hunk 1]
```diff
@@ -83,4 +83,6 @@ public function visit(string $url): void;
     public function getVariantsNames(): array;
 
     public function getOptionValues(string $optionName): array;
+
+    public function getDetails(): string;
```
Comment (Zales0123) [inline src/Sylius/Behat/Page/Shop/Product/ShowPageInterface.php:87]: The tab is called "details" but I believe we're checking a `description` indeed, so we should have `getDescription()` function 🖖

File: src/Sylius/Behat/Page/Shop/Product/ShowPage.php [hunk 2]
```diff
@@ -296,6 +296,10 @@ function (NodeElement $element) {
         );
     }
 
+    public function getDetails(): string
+    {
+        return $this->getDocument()->findAll('css', '[data-tab="details"]')[1]->getText();
+    }
```
Comment (Zales0123) [inline src/Sylius/Behat/Page/Shop/Product/ShowPage.php:302]: Missing blank line below 🚀

**Review by Zales0123 — COMMENTED** [review]
Shouldn't we also serialize the `description` on `ProductTranslation`?

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Tomasz! :1st_place_medal:

**Pull request merged**